src/orb.yml

```diff
@@ -65,22 +65,45 @@ commands:
         description: Optional build command(s)
         type: steps
         default: []
+      yarn:
+        description: Use yarn instead of npm
+        type: boolean
+        default: false
     steps:
       - restore_cache:
           keys:
             - cache-{{ arch }}-{{ .Branch }}-{{ checksum "package.json" }}
       # it would be nice to vary this command based on presence of "package-lock.json"
       # and also allow using yarn or pnpm
-      - run:
-          name: 'Npm CI'
-          command: 'npm ci'
+      - when:
+          condition: << parameters.yarn >>
+          steps:
```

Comment:
ha, I did not know one could add steps into `when` condition like that, this is pretty slick

Reply:
Super powerful.  Also can use empty strings will evaluate to false.